pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -780,12 +780,9 @@ class LocalRunner extends FunctionDetailsCommand {
         @Parameter(names = "--tls-allow-insecure", description = "Allow insecure tls connection #Java, Python")
         protected boolean tlsAllowInsecureConnection;
         // for backwards compatibility purposes
-        @Parameter(names = "--hostname_verification_enabled",
```

Comment:
The backwards compatibility has to break because we are moving `--hostname-verification-enabled` to `arity` 1. As such, users will need to update scripts in order anyway, so it seems like a reasonable time to drop the deprecated parameter.

Note: we have to switch to arity 1 because otherwise JCommander will invert the meaning of the boolean flag and changing the default value will be the opposite of what users expect. See https://jcommander.org/#_boolean